tendrl/monitoring_integration/sync/gluster_cluster_details.py

```diff
@@ -18,45 +18,46 @@ def get_cluster_details(integration_id):
     cluster_details = {}
     try:
         cluster_key = "/clusters/%s" % integration_id
-        cluster_is_managed = etcd_utils.read(
-            cluster_key + "/is_managed"
-        ).value
-        if cluster_is_managed.lower() == "yes":
-            cluster_details["integration_id"] = integration_id
-            # Get node details
-            cluster_details["hosts"] = get_node_details(cluster_key)
-            # Get volume details
-            cluster_details["volumes"] = get_volumes_details(cluster_key)
-            # Get brick details from subvolumes
-            cluster_details["bricks"] = get_brick_details(
-                cluster_details["volumes"],
-                cluster_key
-            )
+        # Get node details
+        cluster_details["hosts"] = get_node_details(cluster_key)
+        # Get volume details
+        cluster_details["volumes"] = get_volumes_details(cluster_key)
+        # Get brick details from subvolumes
+        cluster_details["bricks"] = get_brick_details(
+            cluster_details["volumes"],
+            cluster_key
+        )
     except (etcd.EtcdKeyNotFound, KeyError) as ex:
         logger.log(
             "debug",
             NS.get("publisher_id", None),
             {'message': str(ex)}
         )
-    return cluster_details, constants.GLUSTER_DASHBOARDS
+    return cluster_details
 
 
 def get_node_details(cluster_key):
     node_details = []
     try:
         node_list = utils.get_resource_keys(cluster_key, "nodes")
         for node_id in node_list:
-            node = {}
-            for attr in ATTRS["nodes"]:
-                node[attr] = etcd_utils.read(
-                    cluster_key +
-                    "/nodes/" +
-                    str(node_id) +
-                    "/NodeContext/" +
-                    attr
-                ).value
-            node_details.append(node)
-    except (KeyError, etcd.EtcdKeyNotFound) as ex:
+            try:
+                node = {}
+                for attr in ATTRS["nodes"]:
```

Comment:
There are being taken care as part of marshal/un-marshal to use ClusterNodeContext

Reply:
:+1: